api/client/client_test.go

```diff
@@ -132,9 +132,10 @@ func TestSave(t *testing.T) {
 
 		person := createPerson()
 
-		err = client.Create(context.Background(), dbID, collectionName, person)
+		ids, err := client.Create(context.Background(), dbID, collectionName, person)
 		checkErr(t, err)
 
+		person.ID = ids[0]
```

Comment:
Here's how you now have to update your local model's ID after creating the instance in the db.